doc/source/programs/gdal_contour.rst

```diff
@@ -146,12 +155,112 @@ Examples
 --------
 
 .. example::
-   :title: Creating contours from a DEM
 
-   .. code-block::
+    :title: Creating contours from a DEM
+
+    .. code-block::
+
+        gdal_contour -a elev dem.tif contour.shp -i 10.0
+
+    This would create 10-meter contours from the DEM data in :file:`dem.tif` and
+    produce a shapefile in :file:`contour.shp|shx|dbf` with the contour elevations
+    in the ``elev`` attribute.
+
+.. example::
+
+    :title: Creating polygonal contours from a DEM
+
+    .. code-block:: bash
+
+        $ cat test.asc
+        ncols        2
+        nrows        2
+        xllcorner    0
+        yllcorner    0
+        cellsize     1
+        4 15
+        25 36
+
+        $ gdal_contour test.asc -f GeoJSON /vsistdout/ -i 10 -p -amin min -amax max
+
+    This would create 10-meter polygonal contours from the DEM data in :file:`test.asc`
+    and produce a GeoJSON output with the contour min and max elevations in the ``min``
+    and ``max`` attributes, not including the minimum and maximum values from the raster.
+
```

Comment:
I do not understand. If there is only `-i`, doesn't it mean, that min/max are used just like before the change?
Or maybe I just need another cup of coffee.

Reply:
You a re right:  I removed the `not`.